app/workers/subject_set_subject_counter_worker.rb

```diff
@@ -0,0 +1,19 @@
+class SubjectSetSubjectCounterWorker
+  include Sidekiq::Worker
+
+  sidekiq_options congestion: {
+    interval: 1.hour,
+    max_in_interval: 10,
+    min_delay: 3.minutes,
+    reject_with: :cancel,
```

Comment:
We might want :reschedule? Not entirely sure, which is why a test for expected behaviour might be nice even though it's the gem's job to make it so. I think what we want is for each enqueue to push the job ahead? At the very least, we want to make sure that a job always occurs after the last model was created.